ganttproject/src/main/java/net/sourceforge/ganttproject/task/TaskImpl.java

```diff
@@ -736,9 +736,10 @@ public void setName(final String name) {
         @Override
         public void run() {
           myFieldChange.myEventSender = myPropertiesEventSender;
+          myFieldChange.setOldValue(TaskImpl.this.getName());
           myFieldChange.setValue(name);
           TaskImpl.this.setName(name);
-          if (taskUpdateBuilder != null) {
+          if (taskUpdateBuilder != null && myFieldChange.hasChange()) {
             taskUpdateBuilder.setName(name);
```

Comment:
When a task is created or a user misclicked, the task name gets selected. If you clicked enter after that, a useless update transaction was created